yml/apdb.yaml

```diff
@@ -2758,19 +2775,21 @@ tables:
   '@id': '#DetectorVisitProcessingSummary'
   description: Properties of an image from one processing of a single detector in a visit.
   columns:
-  - name: ccdVisitId
-    '@id': '#DetectorVisitProcessingSummary.ccdVisitId'
-    datatype: long
-    description: Unique identifier.
   - name: visit
     '@id': '#DetectorVisitProcessingSummary.visit'
     datatype: long
+    nullable: false
     description: Id of the visit this visit was a part of.
     mysql:datatype: INTEGER
+    ivoa:ucd: meta.id;obs.image
   - name: detector
     '@id': '#DetectorVisitProcessingSummary.detector'
-    datatype: long
+    datatype: short
+    nullable: false
     description: Id of the detector on the instrument that took this visit.
+      Datatype short instead of byte because of DB concerns about unsigned bytes.
+    ivoa:ucd: meta.id;obs.image
+    # TODO: we probably need to add an index on the above two for this table.
```

Comment:
Primary key would be nice to have too. Are there multiple processings for visit/detector? How are they distinguished here?

Reply:
@ebellm : do we have a plan for multiple processings to go into the same APDB, and if so, how will we distinguish them in `DetectorVisitProcessingSummary`?

Reply:
Maybe we can leave this question for the implementation ticket of `DetectorVisitProcessingSummary`, since we aren't writing anything to it? That would be [DM-39497](https://rubinobs.atlassian.net/browse/DM-39497).

Although, bundling as many schema changes now as we can would be good, it might be better to not block on a table we're not yet using.

Reply:
The table is meant to allow multiple processings of the same raws (that's why it's a _processing_ summary), so there should indeed be a key on processing.  If we go the direction we're thinking on DM-24858, that could be the "release ID" that us used in the ID generators (0 for AP in PP, 1 or 2 for a reprocessing).

Offline/precursor reprocessings go in separate schemas entirely, of course.